docs/examples/repeated_regions.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<body>
+<h1>Repeated Regions Example</h1>
+
+<div class="result">
+    <div class="item">Item1</div>
+    <div class="item">Item2</div>
+    <div class="item">Item3</div>
+    <div class="item">Item4</div>
+</div>
```

Comment:
I think the issue is that this HTML is either misleading or insufficient for the example. There appears (to me) to be a single "result", and within that, multiple "items". A related page object might have `page.results` which returns a list of `Result` regions (a list of one), and each of those could have an `items` property.

It's possible that we could clear up the confusion by renaming the classes. If each "item" is in fact a result, then I would renamed the "result" class to something that suggests it's a container, such as "results" or even "content", and then rename "item" to "result".